src/features/helios/components/potions/component/PotionHouseItems.tsx

```diff
@@ -91,15 +93,24 @@ export const PotionHouseItems: React.FC = () => {
           {Object.values({
             ...POTION_HOUSE_DECORATIONS(),
             ...POTION_HOUSE_ITEMS,
-          }).map((item) => (
-            <Box
-              isSelected={selected.name === item.name}
-              key={item.name}
-              onClick={() => setSelected(item)}
-              image={ITEM_DETAILS[item.name].image}
-              count={inventory[item.name]}
-            />
-          ))}
+          }).map((item) => {
+            if (
+              isExoticCrop(item.name) &&
+              hasFeatureAccess(state, "POTION_SHOP_EXOTIC")
+            ) {
+              return null;
+            }
```

Comment:
```suggestion
            if (
              isExoticCrop(item.name) &&
              !hasFeatureAccess(state, "POTION_SHOP_EXOTIC")
            ) {
              return null;
            }
```

Just a small tweak on the feature flag